package.json

```diff
@@ -37,6 +37,7 @@
   "license": "LGPL 3",
   "scripts": {
     "build": "browserify -d ./src/main.js -p [minifyify --map peaks.min.map --output peaks.min.map] -s peaks -o peaks.min.js",
+    "build-max": "browserify ./src/main.js -s peaks -o peaks.js",
```

Comment:
Also, I didn't intend for that commit to make it to this PR, but it doesn't hurt anything for it to be there ;)